tests/rados/test_bluestoretool_workflows.py

```diff
@@ -672,21 +672,17 @@ def run(ceph_cluster, **kw):
             )
             out = bluestore_obj.show_bluefs_stats(osd_id=osd_id)
             log.info(out)
-            if rhbuild.split(".")[0] >= "7":
-                for pattern in [
-                    "device size",
-                    "DEV/LEV",
-                    "LOG",
-                    "WAL",
-                    "DB",
-                    "SLOW",
-                    "MAXIMUMS",
-                    "TOTAL",
-                ]:
-                    assert pattern in out
-            else:
-                for pattern in ["device size", "wal_total", "db_total", "slow_total"]:
-                    assert pattern in out
+            for pattern in [
```

Comment:
Make the failure graceful, with exception handling for lower releases without the fix.

Reply:
More detailed failure message added